docs/install/docker.md

```diff
@@ -4,7 +4,7 @@ You can install and run Coder using the official Docker images published on [Git
 
 Docker is required. See the [official installation documentation](https://docs.docker.com/install/).
 
-## Run Coder with built-in database and tunnel (quick)
+## Run Coder with the built-in database (quick)
```

Comment:
Let's still mention we're using the tunnel here, both are configured here based on the lack of flags.